### DISCUSSION TITLE
Dịch 'fractionally-strided convolution'

### DISCUSSION THREAD

**Discussion by minhduc0711** [d2l-ai/d2l-vi#2847] [Terminology]
@trGiang99 đang dịch `fractionally-strided` là `sải bước một phần`. Mọi người thấy sao nhỉ?

cc: @lkhphuc @rootonchair @cuongvng @davidnvq @duythanhvn

**Comment by trgiangdo** [comment]
Trong phần [13.10](https://d2l.aivivn.com/chapter_computer-vision/transposed-conv_vn.html) đang dịch là `sải bước phân số`. Sau sẽ cần phải thống nhất 2 chỗ này.

**Comment by rootonchair** [comment]
Mình xin vote là `một phần`
